docker/luna_tutorial/dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:18.04
+
+## supress interactive dialogs
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG USER_ID
+ARG GROUP_ID
+ARG USER_NAME
+ARG GROUP_NAME
+RUN echo $USER_ID:$GROUP_ID $USER_NAME:$GROUP_NAME
+
+## install OS dependencies
+RUN apt-get update
+RUN apt-get install software-properties-common openslide-tools python-openslide default-jre git vim curl tree -y
+RUN add-apt-repository ppa:deadsnakes/ppa
+RUN apt-get install python3.9 python3-pip -y
```

Comment:
merging the RUN statements to 1 through out the dockerfile will reduce the number of layers during docker build.

Reply:
Good Call. Reduced layers.

Reply:
Great! some of the apt-get could be merged too

Reply:
Great! some of the apt-get could be merged too